src/test/scala/fi/oph/koski/raportit/MuuAmmatillinenRaporttiSpec.scala

```diff
@@ -22,7 +22,7 @@ class MuuAmmatillinenRaporttiSpec
   override protected def alterFixture(): Unit = {
     insertMuuAmmatillisenSuorituksenOpiskeluoikeusPäivämäärillä(KoskiSpecificMockOppijat.amis, alkanut = LocalDate.of(2019, 1, 2), päättynyt = LocalDate.of(2019, 12, 31))
     insertMuuAmmatillisenSuorituksenOpiskeluoikeusPäivämäärillä(KoskiSpecificMockOppijat.lukiolainen, alkanut = LocalDate.of(2017, 1, 1), päättynyt = LocalDate.of(2020, 1, 1))
-    insertSisällytettyOpiskeluoikeusSuorituksilla(KoskiSpecificMockOppijat.eero, innerSuoritukset = List(muunAmmatillisenKoulutuksenSuoritus, ammatillinenTutkintoSuoritus(puutarhuri)), outerSuoritukset = List(kiinteistösihteerinMuuAmmatillinenKoulutus()))
```

Comment:
hmm, pitäisiköhän tämäkin validoitumaton testitapaus jotenkin luoda validointisääntöjen ohi poistamisen sijasta? Kun kuitenkin raporteissa oletettavasti edelleen pitää vanhojenkin datojen toimia, vaikka eivät uusien validaatiosääntöjen mukaan olisikaan enää valideja.

Reply:
Mun mielestä tää ei aikasemmin oikein testannut sinänsä mitään. Poistettu päätason suoritus ei ollut muun ammatillisen tyyppiä.

Tavallaan se ehkä testasi sitä, että jos henkilöllä on useampi päätason suoritus, niin lasketaan vain ne jotka ovat tyyppiä "muu", mutta mutta, kun tuo tarkastus alunperinkin tehdään juuri päätason suorituksen tyypillä, niin tuntui aika tarpeettomalta testiltä.

Olemassaolevat fiksturointien etukäteen muuttamiset testeisssä toimii Kosken HTTP-apin yli, ja tässä on just vähän vaikeeta se, että ei meillä taida olla olemassaolevaa tapaa päästä käsiksi sen Kosken sisäisiin funktioihin, jota vasten näitä put/get-opiskeluoikeuksia tehdään.

Sikäli validaatioiden ohittaminen voi olla aika työlästä.